doc/user_guide/eds.rst

```diff
@@ -545,3 +545,125 @@ either with :py:func:`~.misc.material.weight_to_atomic`. The reverse method is :
 
     >>> # With weight_percent from before
     >>> atomic_percent = hs.material.weight_to_atomic(weight_percent)
+
+EDS curve fitting
+-----------------
+
+HyperSpy makes it really easy to extract the intensity of X-ray lines by curve fitting as it is shown in the next example for an EDS-SEM spectrum recorded on a test material EDS-TM001 provided by `BAM <http://www.webshop.bam.de>`_. 
+
+Load the spectrum, define the chemical composition of the sample and set the beam energy.
+
+
+.. code-block:: python
+
+    >>> s = load('bam.msa')
+    >>> s.add_elements(['Al', 'Ar', 'C', 'Cu', 'Mn', 'Zr'])
+    >>> s.set_microscope_parameters(beam_energy=10)
+
+The model is created with :py:func:`~._signals.eds_sem.create_model`. One gaussian is automatically created per X-ray line and a polynomial for the backgronud.
+
+.. code-block:: python
+
+    >>> m = create_model(s)
```

Comment:
Old documentation - should read `s.create_model()`. Same for `hs.load('bam.msa')` higher up